packages/coding-standard/tests/Fixer/Commenting/RemoveUselessClassCommentFixer/Fixture/useless_class_method_comment.php.inc

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Symplify\CodingStandard\Tests\Fixer\Commenting\RemoveUselessClassCommentFixer\Fixture;
+
+class SomeClass
+{
+/**
+ * Get Translator
```

Comment:
Could you add one test case that would be skipped?

E.g.

```php
/**
 * Get special translator
 */
public functoin getTranslator()
{
}
```

And one with space?

```diff
-/**
- * Get normal translator
- */
 public functoin getNormalTranslator()
 {
 } 
```

Reply:
I will try, for "Get normal translator", it seems read next token function name, for "getNormalTranslator" to "Get normal translator" ?

Reply:
Yes, the function name is needed to verify

Reply:
implemented, regex101.com down again so no `@see` comment on new regex constant

Reply:
regex101 up again , add `@see` for new regex constant.